### ISSUE TITLE
Latest Blacklight features: Browse By facet pages

### ISSUE THREAD

**Issue by mcritchlow** [ucsdlib/dams5-cc-pilot#10]
- Can we insert a # per page dropdown as requested in a previous Sprint?
- If this seems possible, please either document what would be needed to implement or, if time permits, create a demo implementation that can be shown at the Sprint Review meeting.

**Comment by mcritchlow** [comment]
Is this in progress? If so, please mark as such.

**Comment by hweng** [comment]
@mcritchlow moved to "in progress".

**Comment by hweng** [comment]
@mcritchlow There is a number of changes from blacklight 4.7.0 to 6.6.2 for facet module. But there is no easy way to insert the # per page dropdown in facet page. It need to update or override the blacklight facet module. 
I think the steps to do:
1. create the facet pages.
2. ask folks in blacklight list for their opinions.
3. implement the # per page dropdown function, update or override the blacklight facet module.

What do you think?

**Comment by mcritchlow** [comment]
@hweng - yes i think that sounds great. I think your second point, in particular, would be interesting. I can't imagine we're the only ones with enough metadata in facets to justify needing that feature.

**Comment by hweng** [comment]
@mcritchlow Continuing with what we talked in meeting, What individual Browse by Facet pages should add to dams5? How about starts with Browse by creator and by topic? We can add more late if the approach is working.

**Comment by mcritchlow** [comment]
@hweng yes I think those two would make the most sense, especially in the context of this ticket. We'll need Format as well, but that wouldn't require the paging functionality since there's such a small fixed list of possible values.

**Comment by hweng** [comment]
@mcritchlow Is Format defined as resource_type_sim in solr?

**Comment by mcritchlow** [comment]
I think in the new CC app, it's either `file_format_sim` or `format_tesim` per this [CC config local](https://github.com/ucsdlib/dams5-cc-pilot/blob/881cd83815a1ef0afadd5d3052bbcb13f7eed9b8/config/locales/curation_concerns.en.yml#L157). Guessing the former since it's nested in the facet section.

@lsitu - does that look right to you?

**Comment by hweng** [comment]
@mcritchlow  I see there is facet field resource_type_sim added in catalog_controller
https://github.com/ucsdlib/dams5-cc-pilot/blob/master/app/controllers/catalog_controller.rb#L38

**Comment by mcritchlow** [comment]
@hweng ah, then it looks like we added our own perhaps. your property is probably correct then :)

**Comment by lsitu** [comment]
@mcritchlow @hweng I think it's different from file format. It's TypeOfResource for resource_type_sim, which is similar to the format facet in dams 4.

**Comment by hweng** [comment]
@lsitu Yes, I think resource_type_sim is similar to the format facet in dams 4. And I checked from log it exist:
(#<CatalogController>)> blacklight_config.facet_fields
=> { "resource_type_sim"=>
  #<Blacklight::Configuration::FacetField label="Type", limit=20, key="resource_type_sim", collapse=true, show=true, if=true, field="resource_type_sim", unless=false> ....}
